packages/viz/package.json

```diff
@@ -1,28 +1,33 @@
 {
   "name": "@carto/toolkit-viz",
-  "version": "0.0.1-rc.13",
+  "version": "0.0.1-rc.16",
   "description": "> TODO: description",
-  "author": "Víctor Velarde <victor.velarde@gmail.com>",
+  "authors": [
+    "Víctor Velarde <victor.velarde@gmail.com>",
+    "Jesús Botella <jbotella@carto.com>"
+  ],
   "homepage": "https://www.carto.com",
   "license": "BSD-3",
-  "browser": "dist/viz.umd.js",
-  "main": "dist/viz.cjs.js",
-  "module": "dist/viz.esm.js",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
   "directories": {
     "lib": "src",
-    "test": "__tests__"
+    "test": "__tests__",
+    "example": "examples"
```

Comment:
Out of curiosity... what's the purpose of directories.example?

Reply:
I reviewed package.json specification and it had that property as well, so I decided to set it to our examples folder. It's not used anywhere (said in docs) but given that we had those two others I decided to fill it in.